Makefile

```diff
@@ -173,6 +174,7 @@ endif
 kustomize:
 ifeq (, $(shell which kustomize))
 	@{ \
+	cd / ;\
```

Comment:
Looks like there is some conflict between versions when trying to install this while in the keda directory 

```
go: sigs.k8s.io/kustomize/kustomize/v3 imports
        sigs.k8s.io/kustomize/kustomize/v3/internal/commands imports
        sigs.k8s.io/kustomize/cmd/kubectl/kubectlcobra imports
        k8s.io/kubectl/pkg/cmd/apply imports
        k8s.io/kubectl/pkg/cmd/util/editor imports
        k8s.io/kubectl/pkg/util/term imports
        github.com/docker/docker/pkg/term imports
        github.com/docker/docker/pkg/term/windows imports
        github.com/Sirupsen/logrus: github.com/Sirupsen/logrus@v1.6.0: parsing go.mod:
        module declares its path as: github.com/sirupsen/logrus
                but was required as: github.com/Sirupsen/logrus
Makefile:175: recipe for target 'kustomize' failed
```


probably wouldn't have happened if you kept the tmp folder hack there. Sorry about that.